openedx/core/djangoapps/enrollments/urls.py

```diff
@@ -29,4 +30,5 @@
         EnrollmentCourseDetailView.as_view(), name='courseenrollmentdetails'),
     url(r'^unenroll/$', UnenrollmentView.as_view(), name='unenrollment'),
     url(r'^roles/$', EnrollmentUserRolesView.as_view(), name='roles'),
+    url(r'^submission_history$', SubmissionHistoryView.as_view()),
```

Comment:
This view should likely also have a `name=` parameter

Reply:
@kaizoku What do you mean by a `name=` parameter?
Right now it accepts a `user=` parameter which is the username. If we add a `name=` parameter should it be user's name?

Reply:
@kaizoku I've updated the view's documentation, hopefully that'll resolve any issues.

Reply:
I mean in the `url()` definition here, like:
```
url(r'^submission_history$', SubmissionHistoryView.as_view(), name='submission_history'),
```

Reply:
@kaizoku Ah, yes, let me add that :)

Reply:
@kaizoku Added a view name.

Reply:
Thanks @xirdneh